webview/src/experiments/components/table/styles.module.scss

```diff
@@ -680,7 +680,7 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
   box-shadow: none;
   position: sticky;
   top: 0;
-  z-index: 4;
+  z-index: 5;
```

Comment:
Noticed yet another small bug that I missed in #2133 😅. Looks like there was an issue with the `z-index`, causing the workspace row to have a higher stacking context than the head tooltips. Fixed by increasing the head `z-index`, making it higher than the workspace row.